### PR TITLE
fix: Refresh not triggered if the access token is not also known, which limits its purpose

### PR DESCRIPTION
**Pull request by cip8**
### 🔗 Linked issue

#890 [second part]

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [x] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

The refresh logic should be triggered if the refresh token is known.

Some backends might require both tokens for a refresh, but this is not default behavior.

Refresh tokens are long-lasting and usually expire after access tokens, so requiring both for a successful refresh should be optional. 

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [ ] I have added tests (if possible).
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by cip8** [comment]
Fix is incomplete - logout doesn't clear refresh cookie so the user re-authenticates on each visit.
Will update soon.

**Comment by cip8** [comment]
Tested with my local setup, everything works fine now.

File: src/runtime/composables/local/useAuth.ts [hunk 1]
```diff
@@ -87,6 +87,28 @@ const signOut: SignOutFunc = async (signOutOptions) => {
...
 
+  // Explicitly clear both cookies to avoid watcher issues
+  // Use the same options to clear the cookie as the ones used to set it
+  useCookie(config.token.cookieName, {
+    maxAge: 0,
+    domain: config.token.cookieDomain,
+    sameSite: config.token.sameSiteAttribute,
+    secure: config.token.secureCookieAttribute,
+    httpOnly: config.token.httpOnlyCookieAttribute
+  }).value = null
```
Comment (phoenix-ru) [inline src/runtime/composables/local/useAuth.ts:98]: I've been trying to understand why is this required - could you please clarify that?

**Comment by einz-loggik** [comment]
@cip8 Whats your status on this? Would love to see this merged because it should fix the problem with automatic logout if the access token reached EOL.

**Comment by cip8** [comment]
Hey @einz-loggik 👋 

Yeah, I’d really like to see this merged as well - but I’ve decided to move on from contributing to this project.

I’ve created my own fork and use it for my projects instead. 

The incredibly long wait times for reviews & some challenges with its maintainers just don't align with how I like to work.

Good luck 💙🖖

File: src/runtime/composables/local/useAuth.ts (hunk 1 above)
Comment (cip8) [reply src/runtime/composables/local/useAuth.ts:98]: coding is hard! for skill issues, use chatgpt
